Sources/Layoutable.swift

```diff
@@ -29,10 +29,23 @@ public protocol Layoutable: AnyObject, Equatable, CustomDebugStringConvertible {
     var superview: PinView? { get }
     var subviews: [PinView] { get }
 
+    /// A Boolean value that determines whether the view is included in the PinLayout's size calculation.
+    ///
+    /// An excluded view does not take up space in the layout
+    /// when using `wrapContent(_:padding:_:)` or `autoSizeThatFits(_:layoutClosure:)`.
+    /// The default value is `true`.
+    var isIncludedInPinLayoutSizeCalculation: Bool { get set }
```

Comment:
Good! Fixed in 7a35f5b11f03cfba84330f9cbf33804296de9ac0.

The property has to still be in `UIView` as an `internal`, because it should still be alive even if the `PinLayout` instance is deallocated.